tools/utils/seed/tasks_tools.ts

```diff
@@ -14,6 +17,33 @@ export function loadTasks(path: string): void {
   readDir(path, taskname => registerTask(taskname, path));
 }
 
+function normalizeTask(task: any, taskName: string) {
+  if (task instanceof Task) {
+    return task;
+  }
+  if (task.prototype && task.prototype instanceof Task) {
+    return new task();
+  }
+  if (typeof task === 'function') {
+    return new class AnonTask extends Task {
+      run(done: any) {
+	if (task.length > 0) {
```

Comment:
hm... here it looks good, not sure if it is a bug in the review view of github or if it is shown nicely here (the same for the other hints about indentation)